### ISSUE TITLE
Book icon Lab messages - Investigate the script to link lab message and sample

### ISSUE THREAD

**Issue by abrudanancuta** [SORMAS-Foundation/SORMAS-Project#11967]
### Problem Description

After executing the script from this ticket https://github.com/hzi-braunschweig/SORMAS-Project/issues/11763 in Production for Nuremberg Health department, it was discovered that second part of the script didn't made any connection back between lab message and sample.

### Proposed Solution

Add to the script as output also the cases that contain the sample with the broken connection to lab message

**Comment by AliciaRaber** [comment]
[2023-05-16_SORMAS_ProjektJF_LaborID.xlsx](https://github.com/hzi-braunschweig/SORMAS-Project/files/11488048/2023-05-16_SORMAS_ProjektJF_LaborID.xlsx)

**Comment by AliciaRaber** [comment]
According to the file the connection has not been established. Feedback from Nuremberg.

**Comment by AliciaRaber** [comment]
@abrudanancuta 
I have checked the results with Traunstein. We surely found out that the archived cases has not been reconnected.
According to the screenshot we have identified one person with two cases. One of them is active, the other is archived.
In person view both samples are visible:
![Person_view_samples](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/91321821/faf3f12e-9d7c-4606-995f-01dbd79a710c)
Then we have case QD2OG7-YMQOCC-UXJLIQ-VHQLSIOQ (also found in the excel for reconnected sample):
![Case_view_active_reconnected](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/91321821/496b7297-010c-4181-b2f3-e0150052cfa2)
In case UHQXOL-L4LB6R-WMGTVR-YDWYKI3U (not found in excel lists) the reconnection did not worked:
![Case_view_archived_not_reconnected](https://github.com/SORMAS-Foundation/SORMAS-Project/assets/91321821/5d754055-1f47-489a-b84e-4b9596e9dd7a)

So we have to edit the script to also include archived cases. Workaround could be that we de-archive the cases temporarily, establishing the script and archive them again.
